bin/zCluster

```diff
@@ -332,7 +338,7 @@ def runOnCatalog(catalog, retriever, retrieverOptions, photoRedshiftEngine, outD
             # Insert a row of data
             c.execute("INSERT INTO redshifts VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", 
```

Comment:
You need to add , ?, ?

as number of parameters doesn't match number of values inserted - again, you would pick this up by running the code.